mappings/net/minecraft/item/Item.mapping

```diff
@@ -18,11 +18,14 @@ CLASS net/minecraft/class_1792 net/minecraft/item/Item
 	METHOD method_19263 isFood ()Z
 		COMMENT Checks if this item is food and therefore is edible.
 	METHOD method_19264 getFoodComponent ()Lnet/minecraft/class_4174;
+		COMMENT @return this item's {@link FoodComponent}, or null if one was not set.
 	METHOD method_21830 getEatSound ()Lnet/minecraft/class_3414;
 	METHOD method_21831 getDrinkSound ()Lnet/minecraft/class_3414;
 	METHOD method_24357 damage (Lnet/minecraft/class_1282;)Z
+		COMMENT @return whether this item can be damaged by the given {@link DamageSource}.
```

Comment:
```suggestion
		COMMENT Returns whether this item can be damaged by the given {@link DamageSource source}.
```
This renders like "Returns whether this item can be damaged by the given `source`"